connaisseur/alert.py

```diff
@@ -48,6 +48,169 @@ def alerting_required(self, event_category: str) -> bool:
         return bool(self.config.get(event_category))
 
 
+class AlertReceiverAuthentication:
+    """
+    Class to store authentication information for securely sending events to the alert receiver.
+    """
+
+    authentication_config: dict = None
+    authentication_scheme: str = None
+
+    class AlertReceiverAuthenticationInterface:
+        def __init__(self, alert_receiver_config: dict, authentication_config_key: str):
+            if authentication_config_key is not None:
+                try:
+                    self.authentication_config = alert_receiver_config[
+                        authentication_config_key
+                    ]
+                except KeyError as err:
+                    raise ConfigurationError(
+                        "No authentication configuration found for dictionary key:"
+                        f"{authentication_config_key}."
+                    ) from err
+
+                self.authentication_scheme = self.authentication_config.get(
+                    "authentication_scheme", self.authentication_scheme
+                )
```

Comment:
What's the second argument here? `self.authentication_scheme` would be `None` at this point, which is the default for `dict.get`.